web/src/scheduled_messages_overlay_ui.js

```diff
@@ -75,6 +75,7 @@ export function remove_scheduled_message_id(scheduled_msg_id) {
 }
 
 export function initialize() {
+
```

Comment:
We should remove this blank line -- I think to always skim individual edited commits after any rebase; I find I introduce this sort of thing all the time when rebasing if I don't.